src/nvim/message.c

```diff
@@ -3472,14 +3470,6 @@ void msg_advance(int col)
     msg_col = col;              // for redirection, may fill it up later
     return;
   }
-  if (ui_has(kUIMessages)) {
-    // TODO(bfredl): use byte count as a basic proxy.
-    // later on we might add proper support for formatted messages.
```

Comment:
I'm not sure if this resolves the todo note, but it does make the logic for ext_messages in msg_advance() redundant. And msg_col is used in some places to determine the formatting for internal messages. Do you remember what you had in mind for "proper support for formatted mesages" @bfredl?

Reply:
Don't remember exactly, maybe stuff like markdown formatting (which implies a different column width). But it is fine to remove this `TODO` regardless, as there is no special case for ext_messages to consider anymore.